test_conformance/gl/CMakeLists.txt

```diff
@@ -1,10 +1,21 @@
+set (MODULE_NAME GL)
+
 if(WIN32)
-list(APPEND CLConform_LIBRARIES  glut32 opengl32 glu32 )
+# Windows MSVC/Intel compiler hits: 
+# - add x64/x86 binaries with glew and glut/freeglut libraries (.lib) to the directory lib/x64 or lib/x86. If names are different then modify list CLConform_LIBRARIES accordingly
+# - GL headers files should be added to project source directory and GL subdirectory
+# - Add .dll files: glut glew to the test binary location.
+    if(CMAKE_CL_64)
+        link_directories("../../lib/x64")
+    else(CMAKE_CL_64)
+        link_directories("../../lib/x86")
+    endif(CMAKE_CL_64)
```

Comment:
Hmm, I'm wondering whether that shouldn't be a CMake macro passed when configuring the build.

Reply:
You mean CMAKE_CL_64 ? I did not add this variable and this is added when cmake project generation by adding CMAKE_CL_64=ON

Reply:
No, I was thinking of the hardcoded relative paths. It would be cleaner to use a CMake variable instead. `CLConform_GL_LIBRARIES_DIR`?